src/main/java/com/mrcrayfish/controllable/client/ControllerInput.java

```diff
@@ -211,6 +213,10 @@ public void onScreenInit(GuiOpenEvent event)
         }
     }
 
+    private double clamp(double min, double max, double value) {
```

Comment:
I suggest removing this and use clamp from MathHelper, a util class in the Minecraft src.

Reply:
Yeah, forgot Minecraft added one.